Flow visibility test on kind fail constantly in old release

The Kind test "E2e tests on a Kind cluster on Linux for Flow Visibility" keep failing when it's run on old release 1.11 and 1.12.
e.g.:https://github.com/antrea-io/antrea/actions/runs/5295909048/jobs/9586763980?pr=5145#logs

The error shows:
```
 2023/06/17 03:11:47 Checking CoreDNS deployment
    fixtures.go:263: Deploying ClickHouse
    flowaggregator_test.go:166: Error when setting up test: services "clickhouse-clickhouse" not found
--- FAIL: TestFlowAggregator (36.55s)
```

Not sure if it's related with image or manifest update, @wsquan171 @heanlan  could you take a look? Thanks.

--- a/test/e2e/framework.go
+++ b/test/e2e/framework.go
@@ -794,10 +794,18 @@ func (data *TestData) deployFlowVisibilityClickHouse() (string, error) {
 	}
 
 	// check clickhouse service http port for service connectivity
-	chSvc, err := data.GetService("flow-visibility", "clickhouse-clickhouse")
-	if err != nil {
-		return "", err
+	var chSvc *corev1.Service
+	if err := wait.PollImmediate(defaultInterval, defaultTimeout, func() (bool, error) {
+		chSvc, err = data.GetService(flowVisibilityNamespace, "clickhouse-clickhouse")
+		if err != nil {
+			return false, nil
+		} else {
+			return true, nil
+		}
+	}); err != nil {
+		return "", fmt.Errorf("timeout waiting for ClickHouse Service: %v", err)
 	}
+
 	if err := wait.PollImmediate(defaultInterval, defaultTimeout, func() (bool, error) {
 		rc, stdout, stderr, err := testData.RunCommandOnNode(controlPlaneNodeName(),
 			fmt.Sprintf("curl -Ss %s:%s", chSvc.Spec.ClusterIP, clickHouseHTTPPort))

